study.md

```diff
@@ -21,18 +21,18 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+<!-- confused about declarations and expressions.  Also feel like I am not on par with the course because I feel like I am not able to write code. -->
 ```
```

Comment:
Keep it up for now, if you still feel this way by later in this week, let us know and we can sit down to chat.